### PR TITLE
Update global.css

### PR DESCRIPTION
**Pull request by okaeiz**
I included the font source because the former code didn't change the RTL font.

## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by wingkwong** [comment]
Can you screenshot the before and after?

File: packages/nc-gui/assets/css/global.css [hunk 1]
```diff
@@ -19,6 +19,8 @@ Apply Vazirmatn for rtl
 
 .rtl .v-application *:not(.material-icons) {
   font-family: Vazirmatn !important;
+  src: url('https://github.com/rastikerdar/vazirmatn/blob/master/fonts/ttf/Vazirmatn-Black.ttf') format('truetype');
```
Comment (pranavxc) [inline packages/nc-gui/assets/css/global.css:22]: Loading font from GitHub is not a good idea

Comment (pranavxc) [reply packages/nc-gui/assets/css/global.css:22]: And we are already loading the font in `assets/style/fonts.css` file

Comment (okaeiz) [reply packages/nc-gui/assets/css/global.css:22]: It does not affect the interface in Windows. The font is not applied on strings.

**Pull request closed without merging**